Homework8/Program.cs

```diff
@@ -0,0 +1,15 @@
+﻿using Homework8.Task1;
+using Homework8.Task2;
+
+namespace Homework8
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+            MassiveElement.ShowMassiveElement();
+            var exceptionGenerator = new ExceptionGenerator();
+            exceptionGenerator.GenerateException();
```

Comment:
Please handle the exception in more appropriate way - use all blocks for handling

Reply:
If I understood correctly - fixed.

Reply:
Missed task condition:
1. Use all blocks
2. Appropriate way - is catching specific exception

Reply:
Done